apps/condo/migrations/20210729162935-0030_alter_forgotpasswordaction_token.js

```diff
@@ -0,0 +1,27 @@
+// auto generated by kmigrator
+// KMIGRATOR:0030_alter_forgotpasswordaction_token:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDMuMi41IG9uIDIwMjEtMDctMjkgMTE6MjkNCg0KZnJvbSBkamFuZ28uZGIgaW1wb3J0IG1pZ3JhdGlvbnMsIG1vZGVscw0KDQoNCmNsYXNzIE1pZ3JhdGlvbihtaWdyYXRpb25zLk1pZ3JhdGlvbik6DQoNCiAgICBkZXBlbmRlbmNpZXMgPSBbDQogICAgICAgICgnX2RqYW5nb19zY2hlbWEnLCAnMDAyOV9tZXJnZV8yMDIxMDcyOV8wODA2JyksDQogICAgXQ0KDQogICAgb3BlcmF0aW9ucyA9IFsNCiAgICAgICAgbWlncmF0aW9ucy5BbHRlckZpZWxkKA0KICAgICAgICAgICAgbW9kZWxfbmFtZT0nZm9yZ290cGFzc3dvcmRhY3Rpb24nLA0KICAgICAgICAgICAgbmFtZT0ndG9rZW4nLA0KICAgICAgICAgICAgZmllbGQ9bW9kZWxzLlRleHRGaWVsZCh1bmlxdWU9VHJ1ZSksDQogICAgICAgICksDQogICAgXQ0K
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;

+--

+-- Alter field token on forgotpasswordaction

+--

+ALTER TABLE "ForgotPasswordAction" ADD CONSTRAINT "ForgotPasswordAction_token_9a0cc9b4_uniq" UNIQUE ("token");

+CREATE INDEX "ForgotPasswordAction_token_9a0cc9b4_like" ON "ForgotPasswordAction" ("token" text_pattern_ops);

+COMMIT;

+
+    `)
+}
+
+exports.down = async (knex) => {

```

Comment:
This method contains incorrect SQL-code, which does't drops added uniq constraint `ForgotPasswordAction_token_9a0cc9b4_uniq`.
It's not enough to drop the index only, I've tested it manually on my local database.

Reply:
@AntonAL Could you please help me, how to solve this?

Reply:
Sure, take a look at [suggested change](https://github.com/open-condo-software/condo/pull/375#discussion_r679722053)

Reply:
Looks, like a bug in `makemigrations` script.
Added a task SBERDOMA-904